test_extract_unique_sensors_01.py

```diff
@@ -1,11 +1,38 @@
 import unittest
+from mock import MagicMock
+from mock import patch
 from extract_unique_sensors_01 import extract_all_sensors
 from extract_unique_sensors_01 import splitting_extracted_sensors
 
 
 class TestExtractSensors(unittest.TestCase):
     """Test class for function test_read_sensors."""
 
+    def test_read_sensors(self):
+        with patch('requests.get') as mock_request:
+            url = "http://lebogang.com"
+            mock_request.return_value.status_code = 200
+            self.assertTrue(url)
+
+        with patch('requests.get') as mock_request:
+            non_existing_url = "http://google.com/fakeurl"
+            mock_request.return_value.status_code = 404
+            self.assertTrue(non_existing_url)
+
+        file_name = "katlego"
+        mock_path = MagicMock()
+        mock_path.Path(file_name).exists().return_value = True
+        self.assertTrue(file_name)
+        self.assertIsInstance(file_name, str)
+        self.assertIsInstance(url, str)
+
+        def Assertlogs(self):
```

Comment:
Not sure why you need a function here. You could just get away with it without a function.

Reply:
Hi @mmphego, this is her attempt, i will look at the case you said she should consider then we can have a chat, I havent had time to talk to her

Reply:
@mmphego There was no need for a function, I see that now. Thanks!